ingest/scripts/compare_hashes.py

```diff
@@ -22,13 +23,102 @@ class Config:
     segmented: str
     nucleotide_sequences: list[str]
     debug_hashes: bool = False
+    slack_hook: str = ""
+
+
+InsdcAccession = str
```

Comment:
@corneliusroemer this is a small refactor to make the code clearer - especially as in a later PR I started handling merging revisions and curations and this gets complicated.